### PR TITLE
IDEX-3674: Add git show command

### PR DESCRIPTION
**Pull request by vinokurig**
@skabashnyuk @vparfonov @sleshchenko  please review

### CODE REVIEW

File: ide/che-core-ide-ui/src/main/resources/org/eclipse/che/ide/ui/smartTree/TreeStyles.css [hunk 1]
```diff
@@ -70,6 +70,11 @@
 
 .selected {
     background-color: projectExplorerSelectedRowBackground;
+    color: projectExplorerJointContainerFill !important;
+}
+
+.selected * {
+    color: projectExplorerJointContainerFill !important;
```
Comment (vzhukovskii) [inline ide/che-core-ide-ui/src/main/resources/org/eclipse/che/ide/ui/smartTree/TreeStyles.css]: Add comment to this block of code, that describe hack, that selected node text forces to apply another color

**Comment by vzhukovskii** [comment]
Other ok

File: ide/che-core-ide-ui/src/main/resources/org/eclipse/che/ide/ui/smartTree/TreeStyles.css (hunk 1 above)
Comment (vinokurig) [inline ide/che-core-ide-ui/src/main/resources/org/eclipse/che/ide/ui/smartTree/TreeStyles.css]: done

File: che-core-git-impl-native/src/main/java/org/eclipse/che/git/impl/nativegit/NativeGit.java [hunk 2]
```diff
@@ -183,6 +184,15 @@ public DiffCommand createDiffCommand() {
     }
 
     /**
+     * @return show command
+     */
+    public ShowCommand createShowCommand() {
```
Comment (vparfonov) [inline che-core-git-impl-native/src/main/java/org/eclipse/che/git/impl/nativegit/NativeGit.java]: Show what? Not good name for class.

Comment (vinokurig) [inline che-core-git-impl-native/src/main/java/org/eclipse/che/git/impl/nativegit/NativeGit.java]: renamed command, response, request and git service method to showFileContent

File: platform-api/che-core-api-git/src/main/java/org/eclipse/che/api/git/GitService.java [hunk 3]
```diff
@@ -215,6 +217,16 @@ public InfoPage diff(DiffRequest request) throws ApiException {
         }
     }
 
+    @Path("show")
```
Comment (skabashnyuk) [inline platform-api/che-core-api-git/src/main/java/org/eclipse/che/api/git/GitService.java]: javadoc

Comment (vinokurig) [inline platform-api/che-core-api-git/src/main/java/org/eclipse/che/api/git/GitService.java]: added a javadoc

**Comment by skabashnyuk** [comment]
ok for me .
 @vparfonov @sleshchenko ?

**Comment by sleshchenko** [comment]
OK for me

**Comment by vparfonov** [comment]
ok

**Pull request merged**